### ISSUE TITLE
Object rest & spread operators break `computed-property-spacing`

### ISSUE THREAD

**Issue by mdziekon** [babel/eslint-plugin-babel#14]
This two lines of code break eslint:
```javascript
const { ...something } = test;
const test = { ...something };
```

with this error (same for both of them):
```javascript
error  There should be no space before '}'  computed-property-spacing
```

It won't break in this situation:
```javascript
const test = {
    ...something
};
```

My .eslintrc file: http://pastebin.com/dBm79tb8

I think there should be an override for this.

**Comment by mdziekon** [comment]
False alarm, I just had to update to the most recent ``babel-eslint`` version (which is 4.1.1 for now).
Sorry for any trouble.
